http.go

```diff
@@ -36,6 +36,11 @@ type ViolationRequest struct {
 	IP string `json:"ip,omitempty"`
 }
 
+const (
+	typeIP    = "ip"
```

Comment:
Maybe if we do this we also update https://github.com/mozilla-services/iprepd/blob/8558ca746cce9c752b753d4bca2f60c49f281781/validators.go#L10-L11 to reference these?

Reply:
yes sir, and at least 20+ instances of "ip" and "email" in the code base. This will come in a follow up PR since there are so many of them

Reply:
👍